config/plugins.js

```diff
@@ -64,5 +64,22 @@ module.exports = ({ env }) => ({
         name: "strapi-prometheus",
       },
     }
-  }
-});
\ No newline at end of file
+  },
+  email: {
+    config: {
+      provider: 'nodemailer',
+      providerOptions: {
+        host: 'smtp.gmail.com',
+        port: 465,
+        auth: {
+          user: "email@gmail.com", // edit here
+          pass: "zzzz zzzz zzzz zzzz", // edit here
```

Comment:
@chungthuang @oriyomibadmus how could we add an environment variable that is accessible in the staging or production environments? Tiago got this working using my credentials to test with, but ideally we could set up the ability to have dev launchers credentials which are not exposed in the source control.

Reply:
Our standard is adding a secret to the kuberentes server by encrpyting it and add to https://github.com/dev-launchers/k8s-infra/tree/main/tenants/production/strapiv4/secrets. The app can access it via an environmental variable, for example, https://github.com/dev-launchers/strapiv4/blob/main/workload/deployment.yaml#L50-L54.

Reply:
Something like:

```
./workload/deployment.yaml

- name: CLIENT_SECRET # replace for 'USER' or/and 'PASSWORD'
    valueFrom:
      secretKeyRef:
        name: auth # replace for 'nodemailer'
        key: discord-client-secret # replace for 'user' or/and 'password'
```

?

Reply:
@tiagopazhs yes we need to add what you said to ./workload/deployment.yaml. And here we can reference the secret through environmental variable by `process.env(USER)`. I would recommending calling it something more specific, like `NODE_MAILER_USER`.

Reply:
> can reference the secret through environmental variable by process.env(USER).
OK

> calling it something more specific, like NODE_MAILER_USER.
Good! I appreciate this suggestion!